<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QMainWindow" name="AppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CP Editor: An editor specially designed for competitive programming</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Tab"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenContest"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionClose_Current"/>
    <addaction name="actionClose_Saved"/>
    <addaction name="actionClose_All"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_indent"/>
    <addaction name="action_unindent"/>
    <addaction name="separator"/>
    <addaction name="action_swap_line_up"/>
    <addaction name="action_swap_line_down"/>
    <addaction name="action_duplicate_line"/>
    <addaction name="action_delete_line"/>
    <addaction name="separator"/>
    <addaction name="action_toggle_comment"/>
    <addaction name="action_toggle_block_comment"/>
    <addaction name="separator"/>
    <addaction name="action_find_replace"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <addaction name="actionCompile"/>
    <addaction name="actionCompile_Run"/>
    <addaction name="actionRun"/>
    <addaction name="actionRun_Detached"/>
    <addaction name="actionKill_Processes"/>
    <addaction name="separator"/>
    <addaction name="actionFormat_code"/>
    <addaction name="actionUse_Snippets"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionEditor_Mode"/>
    <addaction name="actionIO_Mode"/>
    <addaction name="actionSplit_Mode"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="action_reset_settings"/>
    <addaction name="separator"/>
    <addaction name="action_export_settings"/>
    <addaction name="action_import_settings"/>
    <addaction name="separator"/>
    <addaction name="action_export_session"/>
    <addaction name="action_load_session"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Logs"/>
    <addaction name="actionClear_Logs"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="action_report_issues"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionBuildInfo"/>
    <addaction name="actionCheck_for_updates"/>
    <addaction name="actionSupport_me"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuActions"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New File</string>
   </property>
   <property name="toolTip">
    <string>Open a new tab in the editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the file on the disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save as new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="toolTip">
    <string>Save all opened files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Shift+S</string>
   </property>
  </action>
  <action name="actionClose_Current">
   <property name="text">
    <string>Close Current</string>
   </property>
   <property name="toolTip">
    <string>Close current tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose_Saved">
   <property name="text">
    <string>Close Saved</string>
   </property>
   <property name="toolTip">
    <string>Close saved tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+W</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="toolTip">
    <string>Close All the tabs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open File...</string>
   </property>
   <property name="toolTip">
    <string>Open files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpenContest">
   <property name="text">
    <string>Open Contest...</string>
   </property>
   <property name="toolTip">
    <string>Open a Contest</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="action_indent">
   <property name="text">
    <string>Indent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+]</string>
   </property>
  </action>
  <action name="action_unindent">
   <property name="text">
    <string>Unindent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+[</string>
   </property>
  </action>
  <action name="action_swap_line_up">
   <property name="text">
    <string>Swap Line Up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Up</string>
   </property>
  </action>
  <action name="action_swap_line_down">
   <property name="text">
    <string>Swap Line Down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Down</string>
   </property>
  </action>
  <action name="action_duplicate_line">
   <property name="text">
    <string>Duplicate Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="action_delete_line">
   <property name="text">
    <string>Delete Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+K</string>
   </property>
  </action>
  <action name="action_toggle_comment">
   <property name="text">
    <string>Toggle Comment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
  <action name="action_toggle_block_comment">
   <property name="text">
    <string>Toggle Block Comment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+/</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionCompile_Run">
   <property name="text">
    <string>Compile and Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFormat_code">
   <property name="text">
    <string>Format code</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+L</string>
   </property>
  </action>
  <action name="actionRun_Detached">
   <property name="text">
    <string>Run Detached</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+D</string>
   </property>
  </action>
  <action name="actionKill_Processes">
   <property name="text">
    <string>Kill Processes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="action_find_replace">
   <property name="text">
    <string>Find and Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionUse_Snippets">
   <property name="text">
    <string>Use Snippet...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_reset_settings">
   <property name="text">
    <string>Reset Settings</string>
   </property>
  </action>
  <action name="action_export_settings">
   <property name="text">
    <string>Export Settings...</string>
   </property>
  </action>
  <action name="action_import_settings">
   <property name="text">
    <string>Import Settings...</string>
   </property>
  </action>
  <action name="action_export_session">
   <property name="text">
    <string>Export Session...</string>
   </property>
  </action>
  <action name="action_load_session">
   <property name="text">
    <string>Load Session...</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="action_report_issues">
   <property name="text">
    <string>Report issues</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionBuildInfo">
   <property name="text">
    <string>Build Info</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionSupport_me">
   <property name="text">
    <string>Support me</string>
   </property>
  </action>
  <action name="actionEditor_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Editor Mode</string>
   </property>
  </action>
  <action name="actionIO_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>IO Mode</string>
   </property>
  </action>
  <action name="actionSplit_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Split Mode</string>
   </property>
  </action>
  <action name="actionShow_Logs">
   <property name="text">
    <string>Show Log Files</string>
   </property>
  </action>
  <action name="actionClear_Logs">
   <property name="text">
    <string>Delete Log Files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
